docs/Calendar/README.md

```diff
@@ -31,9 +32,16 @@ BpkCalendar(context)
 Example of a calendar controller:
 
 ```Kotlin
-class ExampleBpkCalendarController(private val context: Context) : BpkCalendarController() {
-  override fun onRangeSelected(range: CalendarRange) {
-    // Do something with the selected range
+class ExampleBpkCalendarController(
+  private val context: Context,
+  override val selectionType: SelectionType
+ ) : BpkCalendarController(selectionType) {
+  override fun onRangeSelected(range: CalendarSelection) {
```

Comment:
Thinking of it now I think we should probably change it `onSelected` but we don't need to do it now, we can deprecate this in the future and rename it.